### PR TITLE
Release 0.39

### PR DESCRIPTION
**Pull request by palango**
Create a new release for the testsnets

### CODE REVIEW

File: raiden_contracts/data/deployment_goerli.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
   "chain_id": 5,
```
Comment (palango) [inline raiden_contracts/data/deployment_goerli.json:3]: I'm not sure about the version above. Do we need to change this? It's not set in the 0.37 and 0.38 data dirs as well.

Comment (palango) [reply raiden_contracts/data/deployment_goerli.json:3]: It works with `get_contracts_deployment_info(5, "0.39.0")`

File: RELEASE.rst [hunk 2]
```diff
@@ -90,9 +87,10 @@ We currently deploy on:
 **Checking Validity of deployment_*.json Data**
 
 We have checks for:
-- ``contracts_version``: https://github.com/raiden-network/raiden-contracts/blob/9fd2124eb648a629aee886f37ade5e502431371f/raiden_contracts/deploy/__main__.py#L503
-- ``chain_id``: https://github.com/raiden-network/raiden-contracts/blob/9fd2124eb648a629aee886f37ade5e502431371f/raiden_contracts/deploy/__main__.py#L503
-- deployment addresses & transaction data for each contract, checked against the chain we are running the script on
+
+* ``contracts_version``: https://github.com/raiden-network/raiden-contracts/blob/9fd2124eb648a629aee886f37ade5e502431371f/raiden_contracts/deploy/__main__.py#L503
+* ``chain_id``: https://github.com/raiden-network/raiden-contracts/blob/9fd2124eb648a629aee886f37ade5e502431371f/raiden_contracts/deploy/__main__.py#L503
```
Comment (istankovic) [inline RELEASE.rst:92]: Shouldn't this be pointing to line 504 instead of 503?

**Review by istankovic — APPROVED** [review]

File: raiden_contracts/data/deployment_goerli.json (hunk 1 above)
Comment (istankovic) [reply raiden_contracts/data/deployment_goerli.json:3]: I remember asking Karl about it before when making a release and he said it's just fine to leave it as `null`, nothing should really depend on it.

Comment (palango) [reply raiden_contracts/data/deployment_goerli.json:3]: Cool. good to know!

**Pull request merged**